charts/spire/charts/spire-server/values.yaml

```diff
@@ -82,7 +82,7 @@ dataStorage:
 logLevel: info
 jwtIssuer: oidc-discovery.example.org
 
-clusterName: example-cluster
+clusterName: ""
```

Comment:
I think we should revert this part, so when the chart moves out as subchart all still works the same.